Project/SceneDelegate.swift

```diff
@@ -13,10 +13,17 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
-        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
-        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
+        
         guard let _ = (scene as? UIWindowScene) else { return }
+        
+        let tabBarController = UITabBarController()
+        let navController = UINavigationController(rootViewController: FirstViewController())
+        
+        
+        tabBarController.addChild(navController)
```

Comment:
```suggestion
        tabBarController.setViewControllers
```
только так добавляем